.github/workflows/loan-canary.yml

```diff
@@ -0,0 +1,170 @@
+  
+      - name: Install dependencies
+        run: npm run bootstrap
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.DIGITA_PACKAGES }}
+
+      - name: Build packages
+        run: npm run build:all
+
+
+          
+      - name: Publish packages
+        id: publish-packages
+        run: |
+          npx lerna publish --canary minor --no-verify-access --preid loans --dist-tag loans --yes > .git/canary-release-output
+          cat .git/canary-release-output | grep '=>' | head -n 1 | cut -f 5 -d ' ' | cut -f 1 -d '+' | xargs echo name=version::v | tr -d '[:blank:]' | xargs echo ::set-output
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.DIGITA_PACKAGES }}
+
+  publish-docker:
+
+    name: Publish docker images
+    needs: publish-npm
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CONTAINER_REGISTRY_TOKEN }}
+
+      - name: Build and push manage
+        uses: docker/build-push-action@v2
+        with:
+          context: packages/solid-crs-manage/
+          push: true
+          tags: |
+            ${{ needs.variables.outputs.image_manage }}:loans
+            ${{ needs.variables.outputs.image_manage }}:${{ needs.publish-npm.outputs.version }}
+          build-args: |
+            NPM_TOKEN=${{secrets.DIGITA_PACKAGES}}
+
+      - name: Build and push presentation
+        uses: docker/build-push-action@v2
+        with:
+          context: packages/solid-crs-presentation/
+          push: true
+          tags: |
+            ${{ needs.variables.outputs.image_presentation }}:loans
+            ${{ needs.variables.outputs.image_presentation }}:${{ needs.publish-npm.outputs.version }}
+          build-args: |
+            NPM_TOKEN=${{secrets.DIGITA_PACKAGES}}
+
+      - name: Build and push semcom-node
+        uses: docker/build-push-action@v2
+        with:
+          context: packages/solid-crs-semcom-node/
+          push: true
+          tags: |
+            ${{ needs.variables.outputs.image_semcom_node }}:loans
+            ${{ needs.variables.outputs.image_semcom_node }}:${{ needs.publish-npm.outputs.version }}
+          build-args: |
+            NPM_TOKEN=${{secrets.DIGITA_PACKAGES}}
+
+      - name: Build and push semcom-components
+        uses: docker/build-push-action@v2
+        with:
+          context: packages/solid-crs-semcom-components/
+          push: true
+          tags: |
+            ${{ needs.variables.outputs.image_semcom_components }}:loans
+            ${{ needs.variables.outputs.image_semcom_components }}:${{ needs.publish-npm.outputs.version }}
```

Comment:
Does this version include any `loan(s)` prefix?

Reply:
I think this covers your previous [comment](https://github.com/netwerk-digitaal-erfgoed/solid-crs/pull/570#issuecomment-1056837911) as well?

Reply:
Using tagged images in https://github.com/netwerk-digitaal-erfgoed/infrastructure/commit/59777e541f589ca134d819a753db5a296e3a7178, so this should be good to go now.